doc/nrf/working_with_nrf/nrf70/developing/powersave.rst

```diff
@@ -67,15 +67,20 @@ The following Power Save modes are supported and can be configured by the user o
   nRF70 Series devices operate in this mode by default.
   The user or application can change the device to the Permanent Active mode using the ``NET_REQUEST_WIFI_PS`` network management API.
 
+Further the Wi-Fi® interface can be configured in one of the following dynamic Power Save modes:
+
+* **Normal Power Save:** Delivery Traffic Indication Message (DTIM).
+  DTIM is the default configuration and enabled by the application based on the traffic profile.
+* **Extended Power Save:** Listen interval-based power save mode.
+  The Listen interval based-Power Save mode is supported to enable devices to sleep longer than the DTIM period.
+  It is typically low and not useful for power saving.
+* **Deep sleep:** Target Wake Time (TWT) power save.
+  TWT can be enabled if the connected access point is Wi-Fi 6 capable.
```

Comment:
Formatting is bit off here, Can you please check?

Reply:
Yes, it could be @ajayparida that the second sentence is has too much indentation? And needs to be like the green marked area in this screenshot/same formatting the as the list that starts on line 60.
https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-11243/nrf/_sources/working_with_nrf/nrf70/developing/powersave.rst.txt
![image](https://github.com/nrfconnect/sdk-nrf/assets/51916561/a84a720c-8b58-4cba-bdcf-a5915f207e2c)